amqprs/src/api/tls.rs

```diff
@@ -64,17 +69,16 @@ impl TlsAdaptor {
     /// Panics if private key is invalid.
     pub fn with_client_auth(
         root_ca_cert: Option<&Path>,
-        client_cert: &Path,
-        client_private_key: &Path,
+        client_cert: PathBuf,
+        client_private_key: PathBuf,
```

Comment:
No need to use `PathBuf`, because there is no need to mutate it. 
Also this break public api compatibility. We are serious about the backward-compatibility

Reply:
Fixed by https://github.com/gftea/amqprs/pull/141/commits/8edc20d9c32e6693e7ab8586fc3f5700e0fffaba